### ISSUE TITLE
Add support for Amazon Music

### ISSUE THREAD

**Issue by carlin-q-scott** [carlin-q-scott/browser-media-keys#27]
This might be more challenging than is worthwhile because they don't use a subdomain for their player and people will obviously be doing other things on Amazon.com than listening to music.

**Comment by auipga** [comment]
> they don't use a subdomain for their player

In germany it's https://music.amazon.de. Has the (sub)domain changed globally?

**Comment by carlin-q-scott** [comment]
Hmm, that seems to be available on amazon.com as well. Either this changed recently or they have multiple players. Thanks for asking the question!

Here's the player controls for someone to implement against:
```
<span title="Previous song" class="button previousButton icon-fastBackward" aria-label="Previous song" tabindex="0" role="button"></span>
<span title="Play and pause songs" class="button playerIcon playButton playerIconPause" aria-label="Play and pause songs" tabindex="0" role="button"></span> 
<span title="Play next song" class="button nextButton icon-fastForward" aria-label="Play next song" tabindex="0" role="button"></span>
<span class="trackInfoContainer">...</span>
```

**Comment by carlin-q-scott** [comment]
Implemented with 740f9150f35636c91dca2b9268bb291b905dbfb6

**Comment by carlin-q-scott** [comment]
Too many things match the current xpath for play/pause so this no longer works. Here's a more precise one for pause: **//section[contains(@class,'playbackControlsView')]**//*[contains(@class, 'playerIconPause')].
